appeals/api/src/server/endpoints/documents/documents.controller.js

```diff
@@ -210,20 +213,35 @@ const updateDocuments = async (req, res) => {
 			document.latestVersion = latestDocument?.latestDocumentVersion?.version;
 
 			if (latestDocument && latestDocument.name) {
-				const auditTrail = await createAuditTrail({
-					appealId: appeal.id,
-					azureAdUserId: req.get('azureAdUserId'),
-					details: stringTokenReplacement(AUDIT_TRAIL_DOCUMENT_REDACTED, [
+				if (document.redactionStatus !== latestDocument?.latestDocumentVersion?.redactionStatusId) {
```

Comment:
It looks like this conditional is duplicated

Reply:
good catch, maybe a bad rebase ??